gen/build_deploy/bash.py

```diff
@@ -136,9 +136,9 @@
 
 # check if sort -V works
 function check_sort_capability() {
-    $( echo '1' | sort -V >/dev/null 2>&1 || exit 1 )
+    $( echo '1' | sort -V >/dev/null 2>&1 )
     RC=$?
-    if [[ "$RC" -eq "2" ]]; then
+    if [[ "$RC" -eq "0" ]]; then
```

Comment:
Sorry for delayed response. I was tied up with customer requests. 
I have updatedt he test based on feed back from you guys.
@branden I have used `command` instead of `which`, seems more robust (Also @orsenthil 's SO link re-inforces that)

@orsenthil Yes, I did the review the SO link. Thanks for sharing.